### DISCUSSION TITLE
MacOS 15 Sequoia iPhone mirroring like feature.

### DISCUSSION THREAD

**Discussion by overheat** [rustdesk/rustdesk#8970] [Ideas]
https://mashable.com/article/apple-wwdc-2024-macos-15-announced

Now rustdesk on iOS looks like a viewer but not master(host).
Any plan to support feature like MacOS 15 Sequoia iPhone mirroring?

**Comment by rustdesk** [comment]
I don't think Apple will release this capability to developers.

**Comment by overheat** [comment]
I see.
It's depends on Apple to open such API.
Thanks
